src/components/playback/playbackmanager.js

```diff
@@ -1618,11 +1618,7 @@ class PlaybackManager {
 
             player = player || self._currentPlayer;
             if (player && !enableLocalPlaylistManagement(player)) {
-                if (player.isLocalPlayer) {
-                    return player.seek((ticks || 0) / 10000);
-                } else {
-                    return player.seek(ticks);
-                }
+                return player.seek(ticks);
```

Comment:
Moving back to ticks in the web client would mean only these two sections need custom arithmetic instead of the other ten or twelve instances.

Reply:
The problem is the inconsistency confuses developers and results in broken behavior every so often.

What I would do if I switched to ticks is go and fix the local player so that this code is still not needed. All instances of currentTime should use the same units.

Reply:
To add to this, HTMLMediaElements handle duration and position in seconds, with a double-precision floating point value for milliseconds.

So, from a web client standpoint, it makes a lot more sense to use milliseconds, since that's what everything in the JS ecosystem uses and what JS devs will be familiar with.

Imo, only the values sent and returned from the server (for now) should be in ticks. Everything passed around the web client, as soon as it arrives, should be handled in milliseconds.